apps/landingPage/src/components/Info.js

```diff
@@ -51,15 +53,22 @@ const Container = styled.div`
   flex-direction: column;
   align-items: center;
   padding-top: 80px;
+  @media (max-width: 600px) {
+    padding-top: 20px;
+  }
 `;
 
 const Header = styled.div`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
   align-items: center;
-  height: 250px;
+  height: 250px,
```

Comment:
That might be an issue?

Reply:
Yes! thanks for noticing

Reply:
This is why wrapping styled components is a good idea, and passing it objects instead 😄 

It eliminates the possibility of this error.